src/components/Disputes/panels/AppealPanel.js

```diff
@@ -112,8 +112,9 @@ const AppealPanel = React.memo(function AppealPanel({
         dispute.lastRoundId,
         appealOption.outcome
       )
-      await tx.wait()
+
```

Comment:
are we adding those blank lines between for some reason?

Reply:
Ah yes, i added them for better readability